src/lib/codeSamples/buildRequest.ts

```
@@ -29,43 +29,53 @@ function getHeaders(
+  getAuthorizationsHeaders(authorizations).forEach((value, key) => {
+    resolvedHeaders.set(key, value)
+  })
+
+  return resolvedHeaders
+}
+
+function getAuthorizationsHeaders(authorizations: PlaygroundSecurityScheme | PlaygroundSecurityScheme[]) {
+  const headers = new Headers()
+
   if (authorizations && !Array.isArray(authorizations)) {
     authorizations = [authorizations]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)